Cargo.lock

```diff
@@ -6526,6 +7068,19 @@ version = "1.0.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "ae1a47186c03a32177042e55dbc5fd5aee900b8e0069a8d70fba96a9375cd012"
 
+[[package]]
+name = "sha2"
+version = "0.9.9"
```

Comment:
[Hmph.](https://github.com/paritytech/libsecp256k1/pull/101) I know we don't do a best job of keeping nearcore's dependencies up-to-date ourselves, but this was one merge button away…